frontend-svelte/src/components/widgets/inputs/MenuBarInput.svelte

```diff
@@ -35,15 +35,13 @@
 		// Focus the target so that keyboard inputs are sent to the dropdown
 		(e.target as HTMLElement | undefined)?.focus();
 
-		if (menuListEntry.ref) menuListEntry.ref.isOpen = true;
+		if (menuListEntry.ref) {
+			menuListEntry.ref.isOpen = true;
+			entries = entries;
+		}
 		else throw new Error("The menu bar floating menu has no associated ref");
 	}
 
-	function unFocusEntry(menuListEntry: MenuListEntry, e: FocusEvent) {
-		const blurTarget = (e.target as HTMLElement | undefined)?.closest("[data-menu-bar-input]");
-		if (blurTarget !== self && menuListEntry.ref) menuListEntry.ref.isOpen = false;
-	}
-
```

Comment:
Could you please explain your thoughts behind removing this? Why are you calling it dead code?

Reply:
It's not working. After removing this, it works as intended.

Reply:
I did some further digging in the Vue version and, as far as I can tell, this does seem to be dead code.